### PR TITLE
Use takeLatest

### PR DESCRIPTION
**Pull request by pyjac**
Addresses #1057

### CODE REVIEW

File: app/containers/HomePage/sagas.js [hunk 1]
```diff
@@ -32,9 +33,7 @@ export function* getRepos() {
  * Watches for LOAD_REPOS action and calls handler
```
Comment (mxstbr) [inline app/containers/HomePage/sagas.js]: Let's update this to say "Watches for `LOAD_REPOS` actions and calls `getRepos` when one comes in. By using `takeLatest` (and not `takeEvery`), only the result of the latest API call is applied." (or something like that)

Comment (pyjac) [inline app/containers/HomePage/sagas.js]: Makes sense

Comment (pyjac) [inline app/containers/HomePage/sagas.js]: @mxstbr Done

**Comment by justingreenberg** [comment]
@pyjac @mxstbr one small issue — 1c459e342f41413917bc46f92949be3d4c67f3f6 (removing the `cancel` effect) just needs to be reverted since it will cause regression #384 (fixed in #399) nice work!  :)

**Comment by pyjac** [comment]
@justingreenberg makes sense. How can the `cancel` be covered in the test?

**Comment by mxstbr** [comment]
Ping @justingreenberg

**Comment by justingreenberg** [comment]
@pyjac @mxstbr sorry, the proc cancellation was covered by existing tests, so reverting the removal in b32b6a1 would have restored coverage :) this just needs to be rebased but LGTM 👍

**Comment by mxstbr** [comment]
Awesome, thanks @pyjac!

**Comment by sedubois** [comment]
@mxstbr was it intended to merge to `master`?

**Comment by mxstbr** [comment]
God dammit, my bad. Can you revert this and merge it into dev? Thanks!

**Comment by sedubois** [comment]
@mxstbr I sent a question to GitHub, to know if that can be improved in any way

**Comment by GGAlanSmithee** [comment]
@sedubois you probably already know this, but this can be done by changing the default branch to dev, but this affects everything else too, which probably isn't what you want..

**Comment by sedubois** [comment]
@GGAlanSmithee that's right, we still want people to clone `master` by default.

**Comment by sedubois** [comment]
Response from GitHub: 

> Thanks for your reply.

> We don't have anything that would allow you to use Dev as the compare branch and Master as the clone branch, unfortunately.

> I can understand how this might be useful though, and I will mention it to our team as a Feature Request, but I can't promise that this will be added anytime soon (if at all).

> One thing that might be useful, would be to create a Pull Request Template to remind your users to check that they are using the correct branch:
https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

**Comment by sedubois** [comment]
Created https://github.com/mxstbr/react-boilerplate/pull/1144 to remind to use `dev` branch.

**Pull request merged**